### ISSUE TITLE
Modal needs to update aria-hidden attribute after modal is shown

### ISSUE THREAD

**Issue by joe-watkins** [twbs/bootstrap#19878]
Hello,

V4 Alpha:
It appears that the `aria-hidden="true"` boolean should be updated to `false` once modal fires. This is holding some Assistive Technology back from being able to access the content.

http://v4-alpha.getbootstrap.com/components/modal/

Other than that this modal is really well done! Hats off :)

**Comment by cvrebert** [comment]
@joe-watkins Did you mean "to `false`" instead?

CC: @patrickhlauke

**Comment by pauljadam** [comment]
Need to remove the aria-hidden attribute completely because the dialog content is a child of that aria-hidden element. 

http://pauljadam.com/demos/dialog-DOM.html Need to set aria-hidden on a container that holds all the main content and disable focusability of the main content or some other method of trapping focus in the dialog. Thanks!

**Comment by joe-watkins** [comment]
tnx @cvrebert updated :)

@pauljadam Thanks for finding this issue and helping! ;)

**Comment by patrickhlauke** [comment]
The `aria-hidden` attribute issue is now fixed by https://github.com/twbs/bootstrap/pull/19879

Basic focus trapping is already implemented in the modal. The bigger aspect of having all content outside of the modal wrapped in an `aria-hidden=true` container when modal is visible (already discussed at various points in other related issues) will be handled separately at a later stage (as this will require some more careful refactoring of current markup, CSS and JS)
